### ISSUE TITLE
[Feature Request] "Sweet Shadows" around blockquotes

### ISSUE THREAD

**Issue by DutchPete** [Spekulucius/obsidian-brainhack#6]
Live Preview should be as close as possible to Reading mode. Blockquotes in the former are still missing.

It is not a big deal, merely a "nice to have" 😀

**Comment by Spekulucius** [comment]
I know, and tried - unfortunately, blockquotes get separated by line in preview mode, so adding shadows would cause them to overlap.

since they also don't have a shared parent separated from other content, it is also not possible (afaik) to target the last blockquote line per-blockquote.

it is still a thing and bugging me as well, but atm, it is too much effort fixing this - maybe I'll look into this in the future, but as of now, I scrapped the idea of implementing blockquote shadows in preview mode. I'm sorry :/

**Comment by DutchPete** [comment]
I have good news for you! It IS possible to have a shadow around blockquotes in Live Preview. I found this in a snippet I had:

```
.markdown-source-view.mod-cm6.is-live-preview .HyperMD-quote, div:not(.cm-embed-block) > blockquote {
   box-shadow: 2px 3px 20px #4d4d4d;
}
```

I literally added the code above in a snippet for Brainhack, and here is the result:

![image](https://github.com/user-attachments/assets/7712e653-574e-46d5-94ac-6508b494b8e5)
Like I said, from my side it is not a big deal. But since I found this, I thought I'd share it with you to save you spending time scouting around for a solution.

What would also be nice is if transcluded/embedded blockquotes look the same as the original, instead of like this:

![image](https://github.com/user-attachments/assets/a4ba8bcd-12df-47cd-9d6e-e407240fe646)
though I don't know if that is possible.
